mop/lib/mop/internal/util.pm

```diff
@@ -246,6 +246,7 @@ sub INSTALL_FINALIZATION_RUNNER {
             # an eval STRING;
             || (caller(3))[3] eq '(eval)';
 
+    # Any reason to use private API instead of public here?
```

Comment:
I am sure I had a reason at one point, but I don't recall it now, so, no, it should use the public API.